docs/index.md

```diff
@@ -2,11 +2,41 @@
 layout: home
 ---
 
-Build, manage and test your [Auth0](http://auth0.com/) integrations from the command line.
+Build, manage and test your [Auth0](https://auth0.com/) integrations from the command line.
 
 ## Installation
 
-Installation instructions available in [project README](https://github.com/auth0/auth0-cli#installation).
```

Comment:
It felt a bit odd to redirect to the README for the install but not for the authentication part so I ended up bringing it over here as well so users do't have to go back and forth between the github repo and the dedicated docs site.